third_party/blink/renderer/modules/webaudio/base_audio_context.cc

```diff
@@ -101,6 +101,13 @@ BaseAudioContext::BaseAudioContext(Document* document,
       periodic_wave_triangle_(nullptr) {}
 
 BaseAudioContext::~BaseAudioContext() {
+  if (recordreplay::AreEventsDisallowed() &&
+      recordreplay::IsRecordingOrReplaying("leak-references",
+                                           "~BaseAudioContext")) {
```

Comment:
Just call AreEventsDisallowed and pass in a subfeature.  There isn't a need to use two separate feature tests here.

Reply:
I'm just going with what I have learned previously. I guess I have not considered using `AreEventsDisallowed(const char* why)` before. Changing it now.